README/Design/Decidability.agda

```diff
@@ -37,22 +37,20 @@ ex₁ : (n : ℕ) → Reflects (zero ≡ suc n) false
 ex₁ n = ofⁿ λ ()
 
 ex₂ : (b : Bool) → Reflects (T b) b
-ex₂ false = ofⁿ id
-ex₂ true  = ofʸ tt
+ex₂ = T⁺
```

Comment:
I'm not keen about these changes to the README file. The whole point was to demonstrate the record fields `does` and `proof` (especially in the second example) but they no longer exist here.

Reply:
Yes, I see. Those changes aren't difficult to revert, and would only amount to redundant repetition. But I found myself wanting to 'promote' the definitions from (mere) 'examples' under `README.*` to *bona fide* definitions under `src.*`... if you see what I mean?

Reply:
Now reverted.